pkg/orchestrator/etcd_worker.go

```diff
@@ -153,17 +158,11 @@ func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session,
 			return cerrors.ErrEtcdSessionDone.GenWithStackByArgs()
 		case <-ticker.C:
 			// There is no new event to handle on timer ticks, so we have nothing here.
-			if time.Since(lastReceivedEventTime) > etcdRequestProgressDuration {
-				if err := worker.client.RequestProgress(ctx); err != nil {
-					log.Warn("failed to request progress for etcd watcher", zap.Error(err))
-				}
-			}
```

Comment:
Does it necessary to keep this ticker branch?

Reply:
Now we call the `RequestProgress`  in `client.go`:https://github.com/pingcap/ticdc/pull/3667/files#diff-b8cb0ded4b02fc94e0aae4d43ec9f57c392846dde4642587fd5423426713f080R233

Reply:
I mean the ticker can be removed

Reply:
Got it, but the branch is still needed becasue we need it to go through the select block and  tick `reactor`: https://github.com/pingcap/ticdc/pull/3667/files#diff-1f744cd3130fdaa4293a04fa689c341aa1086bc64278c7a5af46e7774dc16cd9R217

Reply:
got